internal/command/testdata/login-oauth-server/oauthserver.go

```diff
@@ -16,9 +16,9 @@ import (
 // Handler is an implementation of net/http.Handler that provides a stub
 // OAuth server implementation with the following endpoints:
 //
-//     /authz  - authorization endpoint
-//     /token  - token endpoint
-//     /revoke - token revocation (logout) endpoint
+//	/authz  - authorization endpoint
```

Comment:
I believe this want unintended.

Reply:
It's the result of running `gofmt` and `goimport` in my local env 🤔

Reply:
Ok, let's leave it then.